source/lib-intro.tex

```diff
@@ -900,14 +900,19 @@
 \end{footnote}
 
 \pnum
-Whenever a name \tcode{x} defined in the standard library is mentioned,
-the name \tcode{x} is assumed to be fully qualified as
-\tcode{::std::x},
-unless explicitly described otherwise. For example, if the \effects element
-for library function \tcode{F} is described as calling library function \tcode{G},
-the function
-\tcode{::std::G}
-is meant.
```

Comment:
This part is already being removed by LWG 2818 so I think this editorial change would be clearer with just the addition below (possibly rebased on 2818).